### PR TITLE
testing pr reviews

### PR DESCRIPTION
**Pull request by k0shk0sh**

### CODE REVIEW

**Review by k0shk0sh — COMMENTED** [review]
Test

File: branch_file [hunk 1]
```diff
@@ -1 +1,2 @@
 Hello World
```
Comment (k0shk0sh) [inline branch_file]: Hello world

Comment (FastAccess) [inline branch_file]: Change

Comment (FastAccess) [inline branch_file]: Hello

Comment (FastAccess) [inline branch_file]: Change

File: branch_file [hunk 2]
```diff
@@ -1 +1,2 @@
...
+Testing PR Reviews
```
Comment (k0shk0sh) [inline branch_file]: Yy

**Pull request closed without merging**